webpack/stories/docs/APIMiddleware.md

```diff
@@ -1,48 +1,59 @@
 # API Middleware
 
 Instead of each component handling API calls in the same way we have the API Middleware that will handle it instead.
-Current use as an action:
 
-API_OPERATIONS is defined in 'webpack/assets/javascripts/react_app/redux/API.js'
-or 'foremanReact/redux/API' for plugins
+the API Actions can be imported from 'webpack/assets/javascripts/react_app/redux/API/index.js'
+or 'foremanReact/redux/API/index.js' for plugins
+
+currently the `get` action is available, and soon we will support all of the API operations.
+
+## How to use it in your action
+
 ```js
-import { API_OPERATIONS } from ...;
-const someAction = url => ({
-  type: API_OPERATIONS.GET,
-  key: COMPONENT_NAME,
-  url,
-});
+import { get } from '../../redux/API';
+
+const someAction = url =>
+  get({
+    key: MY_SPECIAL_KEY, // this will be used later to identify your API call, so keep it unique.
+    url: url,
+    params: {} // some params you will want to pass to the API request.
```

Comment:
Currently the middleware uses ` payload.params || {}` and not params. is it going to change?

Reply:
As discussed in https://github.com/theforeman/foreman/pull/7308#discussion_r362395295 after the technical debt of this components will be solved, I would like everything to go inside the payload, and remove the current option to pass your own payload, although when using the action creator the consumer shouldn't care much how it is implemented.

Reply:
I agree that the user should not care about the implementation, but should it change in this PR?
or should the `get` function do
``` JS 
payload: {
...payload, params
}
```
?

Reply:
Yes I was thinking about something like that.. will update soon this PR